projects/serializers.py

```diff
@@ -0,0 +1,55 @@
+from rest_framework import serializers
+
+from projects.models import Project
+
+class ProjectListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Project
+        fields = (
+            'id',
+            'name',
+            'status',
+            'information',
+            'required_funding',
+            'raised_amount',
+            'company',
+        )
+        
+
+class ProjectCreateUpdateSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Project
+        fields = (
+            'id',
+            'name',
+            'status',
+            'information',
+            'required_funding',
+            'raised_amount',
+        )
+        extra_kwargs = {
+            'company': {'required': False}
+        }
+
+    def validate(self, attrs):
+        required_funding = attrs.get("required_funding")
+        raised_amount = attrs.get("raised_amount")
+        
+        if required_funding < 0:
+            raise serializers.ValidationError("Required funding cannot be negative.")
+        if raised_amount < 0:
+            raise serializers.ValidationError("Raised amount cannot be negative.")
+        if raised_amount > required_funding:
+            raise serializers.ValidationError("Raised amount cannot be greater than required funding.")
```

Comment:
If `raised_amount` is not provided, it could cause a `TypeError` on comparison with `required_funding`.